docs/blog/2020-02-18-gazette-2019-recap/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Gatsby Gazette: The 2019 Recap"
+date: 2020-02-18
+author: Sidhartha Chatterjee
+excerpt: "The last year has been an exciting time for Gatsby. Let's recap over everything we did in 2019."
+tags:
+  - gazette
+  - community
+---
+
+Welcome to a _special_ edition of the Gatsby Gazette. This one will recap everything we did in 2019. Well, not _everything_. But everything major enough that we can fit into this post.
+
+For our more frequent readers who might already know much of this, we'll follow up with a walkthrough of our plans for 2020.
+
+Before we dive in, thank you, dear reader. If you've been contributing code to Gatsby, helping improve our documentation, reporting bugs and issues or even simply using it, you've enabled so much of the progress we've made and will continue to make this year. Our superpower is _you_, the Gatsby community.
+
+## The 2019 Recap
+
+Let's get into it then.
+
+1348 folks started contributing to Gatsby in 2019.
+
+Together, we made 5657 commits to Gatsby in 2019.
+
+That's a 29% increase from 2018 (4364) and a 153% increase from 2017 (2236).
```

Comment:
I don't think this needs a line break? (If this was a list I'd expect bullet points instead)

Reply:
Good point